CataclysmWin.cbp

```diff
@@ -7,170 +7,309 @@
 		<Option compiler="gcc" />
 		<Build>
 			<Target title="Release">
-				<Option output="Cataclysm" prefix_auto="1" extension_auto="1" />
+				<Option output="CataclysmWin" prefix_auto="1" extension_auto="1" />
```

Comment:
I'll take a look: as I read that code, the executable would be "CataclysmWin" rather than Cataclysm, and that seemed off..